.github/workflows/ci-build.yml

```diff
@@ -73,8 +73,11 @@ jobs:
           path: build.log
 
       - name: Verify Changed Files
-        uses: tj-actions/verify-changed-files@v20
         id: verify-changed-files
+        run: |
+          set -o pipefail
+          changed_files=$(echo -n "$(git diff --name-only HEAD 2>/dev/null ; git ls-files --others --exclude-standard 2>/dev/null)"|tr '\n' ' ')
```

Comment:
Can you use `&&` instead of `;` so the exit code is not ignored?
Is it really necessary to redirect stderr to /dev/null? It might hide the reason why commands fail and I think only stdout is assigned based on the output.

Reply:
Done.
I added `dev/null` because saw one error message during my experiments on the command line.
But I agree, let's go without.